package.json

```
@@ -23,8 +23,8 @@
     "build:production": "env-cmd -f .env.production react-scripts build",
     "predeploy": "npm run build:staging",
     "deploy": "gh-pages -d build",
-    "eslint": "eslint --ext .js --no-color --fix .",
-    "prettier": "prettier --write .",
+    "eslint": "eslint --ext .js --no-color --fix ",
+    "prettier": "prettier --write ",
```

Reverting this change.
```suggestion
    "eslint": "eslint --ext .js --no-color --fix .",
    "prettier": "prettier --write .",
```